library/stdtypes.po

```diff
@@ -1847,9 +1846,8 @@ msgid "``s.remove(x)``"
 msgstr "``s.remove(x)``"
 
 #: ../Doc/library/stdtypes.rst:1083
-#, fuzzy
 msgid "remove the first item from *s* where ``s[i]`` is equal to *x*"
-msgstr "supprime le premier élément de *s* pour qui ``s[i] == x``"
+msgstr "supprime le premier item de *s* pour lequel ``s[i] == x``"
```

Comment:
élément ça me plaisait bien.

Reply:
``s[i]`` is equal to *x*" à été modifié, il faut donc le traduire en ``s[i]`` est égal à ``x``

Reply:
Double rectification effectuée.